public/css/archetype-icons.css

```
@@ -0,0 +1,175 @@
+/* ============================================ */
+
+.archetype-button::after {
+    content: attr(aria-label);
+    position: absolute;
+    bottom: -40px;
+    left: 50%;
+    transform: translateX(-50%) scale(0);
+    padding: 0.5rem 1rem;
+    background: rgba(26, 10, 46, 0.95);
+    color: var(--neon-pink);
+    font-family: 'Orbitron', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)